Apps/Medication/src/Services/RestMedicationService.cs

```diff
@@ -78,37 +83,51 @@ public async Task<string> GetPatientPHN(string hdid)
         }
 
         /// <inheritdoc/>
-        public async Task<List<Prescription>> GetPrescriptionsAsync(string phn, string userId, string ipAddress)
+        public async Task<List<MedicationStatement>> GetMedicationsAsync(string phn, string userId, string ipAddress)
         {
-            //ServicePointManager.ServerCertificateValidationCallback += (sender, cert, chain, sslPolicyErrors) => true;
-            HttpClientHandler clientHandler = new HttpClientHandler();
-            clientHandler.ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) => { return true; };
-
-            using (HttpClient client = new HttpClient(clientHandler))
+            JWTModel jwtModel = this.AuthenticateService();
+            using (HttpClientHandler clientHandler = new HttpClientHandler())
             {
-                client.DefaultRequestHeaders.Accept.Clear();
-                client.DefaultRequestHeaders.Accept.Add(
-                    new MediaTypeWithQualityHeaderValue(MediaTypeNames.Application.Json));
-
-                // Inject JWT
-                // client.DefaultRequestHeaders.Add("Authorization", "Bearer ${TOKEN}")
-                string hnClientUrl = this.configService.GetSection("HNClient").GetValue<string>("Url");
-                HNMessage responseMessage;
+                clientHandler.ServerCertificateCustomValidationCallback = (sender, cert, chain, sslPolicyErrors) => { return true; };
 
-                HNMessage requestMessage = this.medicationParser.CreateRequestMessage(phn, userId, ipAddress);
-                HttpResponseMessage response = await client.PostAsJsonAsync(new Uri($"{hnClientUrl}v1/api/HNClient"), requestMessage).ConfigureAwait(true);
-                if (response.IsSuccessStatusCode)
-                {
-                    string payload = await response.Content.ReadAsStringAsync().ConfigureAwait(true);
-                    responseMessage = JsonConvert.DeserializeObject<HNMessage>(payload);
-                }
-                else
+                using (HttpClient client = new HttpClient(clientHandler))
```

Comment:
I have a PR that will change the way we create the HttpClient (basically because of our unit test mocks)